raiden/network/rpc/client.py

```diff
@@ -126,7 +127,7 @@ def format_data_for_call(
         to: address = b'',
         value: int = 0,
         data: bytes = b'',
-        startgas: int = GAS_PRICE,
+        startgas: int = GAS_LIMIT,
```

Comment:
I find this a rather unintuitive name, should we rename it to something like `initial_gas_limit` once we're at it?